cura/CuraApplication.py

```diff
@@ -455,7 +453,7 @@ def startSplashWindowPhase(self) -> None:
             # Misc.:
             "ConsoleLogger", #You want to be able to read the log if something goes wrong.
             "CuraEngineBackend", #Cura is useless without this one since you can't slice.
-            "UserAgreement", #Our lawyers want every user to see this at least once.
+            # NOTE: User-Agreement is part of the 'onboarding flow' now (See Welcome Pages).
```

Comment:
This message will be useless after a while because it doesn't matter that this used to be a plug-in at some point.

Reply:
Removed.